SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -1950,14 +1950,29 @@ ERPCResult USpatialReceiver::ApplyRPCInternal(UObject* TargetObject, UFunction*
 	TSharedPtr<FRepLayout> RepLayout = NetDriver->GetFunctionRepLayout(Function);
 	RepLayout_ReceivePropertiesForRPC(*RepLayout, PayloadReader, Parms);
 
-	if ((UnresolvedRefs.Num() == 0) || bApplyWithUnresolvedRefs)
+	if (UnresolvedRefs.Num() != 0 && !bApplyWithUnresolvedRefs)
 	{
-		TargetObject->ProcessEvent(Function, Parms);
-		Result = ERPCResult::Success;
+		ErrorInfo.ErrorCode = ERPCResult::UnresolvedParameters;
 	}
 	else
 	{
-		Result = ERPCResult::UnresolvedParameters;
+		// Get the supposedly net owned actor
```

Comment:
NIT, might be misunderstanding this comment, but personally think this wasn't explicit enough before?

```suggestion
		// Get the RPC target Actor.
```